Makefile

```diff
@@ -1,16 +1,16 @@
 .PHONY: all binary ut clean
 
 BUILD_DIR=build_calico_rkt
-BUILD_FILES=Dockerfile requirements.txt
+BUILD_FILES=$(BUILD_DIR)/Dockerfile $(BUILD_DIR)/requirements.txt
```

Comment:
When I created the calico-rkt repo I fixed that; the build_calico_rkt directory has been added back in at some point since then. I think you should back it out (see Tom's libnetwork-plugin for supporting evidence that it's not necessary for plugins).